Bojzen-I-Mitten/Stort-spelprojekt
# Gui/scale f ix

GUI now scales with the size of the window.

To test:
- Should work with any scene. Use DemoScene
- Test resize the window and see so that the UI scales.
